Pdf/Helper/Data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace MagentoJapan\Pdf\Helper;
+
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\App\Helper\Context;
+
+/**
+ * Class Data
+ * @package MagentoJapan\Pdf\Helper
+ */
+class Data extends \Magento\Framework\App\Helper\AbstractHelper
+{
+
+    /**
+     * font file path
+     */
+    const FONT_DIR = 'lib/fonts/';
+
+    /**
+     * @var string
+     */
+    protected $_dir;
```

Comment:
Adding underscore to private properties is more or less outdates convention. It's safe to use names without it.